contents/docs/self-host/deploy/hosting-in-eu.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Hosting in EU
+sidebar: Docs
+showTitle: true
+---
+
+EU users with stringent data control requirements may not be able to host with US-based companies, even if the servers are physically located within the EU. This list of providers is a good place to start if you fall into this camp. The following are all EU-based companies. 
+
+- [Leaseweb](https://www.leaseweb.com)
```

Comment:
Note: we currently do not officially support any of those.

Reply:
@guidoiaquinti - I'm assuming that customers here would be falling under the 'Other Kubernetes' deployment targets documented here https://posthog.com/docs/self-host/deploy/other